### DISCUSSION TITLE
geomap auto zoom

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana#50527] [Plugins]
Greetings,

If we have a list of cities to choose from as drop down from variables, it is possible for the map to zoom in to selected city?

Thanks!

**Comment by nmarrs** [comment]
Hi @yosiasz, thanks for the suggestion. I believe that this use case would be covered by an existing feature request that we currently have on our backlog for Geomap

https://github.com/grafana/grafana/issues/40496 - Auto zoom feature request

https://github.com/grafana/grafana/issues/44240 - Geomap backlog

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
